src/lib/updating-element.ts

```diff
@@ -288,35 +270,18 @@ export abstract class UpdatingElement extends HTMLElement {
     // metadata.
     this._ensureClassProperties();
     this._classProperties!.set(name, options);
-    if (!options.noAccessor) {
-      const superDesc = descriptorFromPrototype(name, this.prototype);
-      let desc;
-      // If there is a super accessor, capture it and "super" to it
-      if (superDesc !== undefined && (superDesc.set && superDesc.get)) {
-        const {set, get} = superDesc;
-        desc = {
-          get() { return get.call(this); },
-          set(value: any) {
-            const oldValue = this[name];
-            set.call(this, value);
-            this.requestUpdate(name, oldValue);
-          },
-          configurable : true,
-          enumerable : true
-        };
-      } else {
+    if (!options.noAccessor && !this.prototype.hasOwnProperty(name)) {
```

Comment:
```suggestion
    // Do not generate an accessor if the prototype already has one, since
    // it would be lost otherwise and that would never be the user's intention;
    // Instead, we expect users to call `requestUpdate` themselves from
    // user-defined accessors. Note that if the super has an accessor we will
    // still overwrite it
    if (!options.noAccessor && !this.prototype.hasOwnProperty(name)) {
```